packages/core/src/components/ozon-content/nodes/lijst.node.tsx

```diff
@@ -0,0 +1,41 @@
+import { h, Fragment } from '@stencil/core';
+
+import { getNodeName } from '../get-node-name.function';
+import { OzonContentNodeContext } from '../ozon-content-node-context.interface';
+import { OzonContentNode } from '../ozon-content-node.interface';
+
+export class OzonContentLijstNode implements OzonContentNode {
+  name = 'Lijst';
+
+  render(node: Element, { mapNodeToJsx }: OzonContentNodeContext) {
+    const childNodes = Array.from(node.childNodes);
+    const aanhef = childNodes.find(n => getNodeName(n) === 'Lijstaanhef')?.textContent;
+    const sluiting = childNodes.find(n => getNodeName(n) === 'Lijstsluiting')?.textContent;
+    const listItems = childNodes.filter(n => getNodeName(n) === 'Li');
+
+    return (
+      <div class="dso-ozon-lijst od-Lijst">
+        {aanhef && <p class="od-Lijstaanhef">{aanhef}</p>}
+        <ul class={node.getAttribute('type') ?? ''}>
+          {listItems.map(item => {
+            const itemNodes = Array.from(item.childNodes);
+            const lijst = itemNodes.find(n => getNodeName(n) === 'Lijst')
+            const liNummer = itemNodes.find(n => getNodeName(n) === 'LiNummer')?.textContent;
+            const al = itemNodes.find(n => getNodeName(n) === 'Al');
+
+            return (
+              <>
+                <li class="od-Li">
+                  <span class="od-LiNummer">{liNummer}</span>
+                  {lijst && mapNodeToJsx(lijst)}
+                  {al && mapNodeToJsx(al)}
```

Comment:
We kunnen beter _alle_ childnodes doormappen.

Reply:
Ja handig.